map/src/context/TracksManager.js

```diff
@@ -320,26 +320,29 @@ function prepareTrack(track, localName = null, originalName = null) {
 }
 
 function hasGeo(track) {
+    let hasGeo;
```

Comment:
forgotten default value: let hasGeo = false;

Also, need to rewrite whole function to avoid forEach().
Just use for() and return true if geo exists
(in this case we can drop hasGeo flag)